<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>731</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="feedBtn">
       <property name="text">
        <string>记录我的投喂</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="catinfoBtn">
       <property name="text">
        <string>猫猫花名册</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="witnessBtn">
       <property name="text">
        <string>猫猫位置打卡</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="newAdminBtn">
       <property name="text">
        <string>新增管理员</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>981</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="catTabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>701</width>
        <height>551</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <column>
       <property name="text">
        <string>猫猫编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>猫猫名字</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>性别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>品种</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>常出现地点</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>疫苗状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>备注</string>
       </property>
      </column>
      <column/>
      <column/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>猫名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addCatNameEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="addCatGenderBox">
         <item>
          <property name="text">
           <string>公</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>母</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>品种：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="AddCatRaceEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>530</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>作为管理员，请细心填写~</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="addLocBox">
         <item>
          <property name="text">
           <string>荔园1~5栋</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>荔园6~10栋</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>主楼以及活动中心</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T3~T5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T2、T6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>运动场</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>研究生教学区</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>常出现地点：</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>疫苗状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="addCatVacBox">
         <item>
          <property name="text">
           <string>未接种</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>已接种</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>备注：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="addOtherEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addCatButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>新增猫猫</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="witnessTable">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>511</width>
        <height>541</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>打卡编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>猫猫名字</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>地点</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>打卡用户</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>打卡时间</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>打卡地点：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>猫猫名字：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="witCatNameBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addWitButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QComboBox" name="witLocBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>荔园1~5栋</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>荔园6~10栋</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>主楼以及活动中心</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T3~T5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T2、T6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>运动场</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>研究生教学区</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="feedTable">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>601</width>
        <height>561</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column>
       <property name="text">
        <string>投喂编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>猫猫名字</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>食物</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>投喂地点</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>投喂用户</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>投喂时间</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>投喂猫猫：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>注意，为了猫猫的健康，请勿喂食自带食物</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>投喂务必领取提供的专用食物，谢谢</string>
      </property>
     </widget>
     <widget class="QComboBox" name="feedCatBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>投喂食物：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="feedFoodBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>猫粮1号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>猫用奥利奥</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>猫用饼干</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>喵比克</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>投喂地点：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addFeedBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QComboBox" name="feedLocBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>319</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>荔园1~5栋</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>荔园6~10栋</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>主楼以及活动中心</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T3~T5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T2、T6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>运动场</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>研究生教学区</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
